config/locales/sv.yml

```diff
@@ -1054,7 +1057,15 @@ sv:
         message_text:
           fee_is_unpaid: Observera att ditt företags H-märkespremie inte är betald.
           how_to_pay_fee: Betala den ett år till via länken nedan. Detta gör att du fortsätter synas på kartan och fortsätter ha ett H-märkt företag.
-          company_link_login_msg: " (du kan behöva logga in först innan du ser knappen för att betala)."
+          company_link_login_msg: &co_link_login " (du kan behöva logga in först innan du ser knappen för att betala)."
+
+      co_info_incomplete:
+        subject: Please complete the company information
+        greeting: *email_greeting
+        message_text:
+          complete_the_info: "You need to complete the info for %{co_name}. If you don't it won't show up on the map or in google searches"
```

Comment:
Q: What happens if name is empty? 

Du behöver färdigställa informationen om %{co_name}. Om du inte gör det syns du inte i sökningar. Saknar du adress syns du heller inte på kartan.

Reply:
If the company name is blank, then it shows` company_needs_name: "Org nummer %{co_number}"`

where `%{co_name}` is.  (That logic is in the mailer view app/views/member_mailer/company_info_incomplete.html.haml)

So it would say:

Du behöver färdigställa informationen om Org nummer %{co_number}. Om du inte gör det syns du inte i sökningar. Saknar du adress syns du heller inte på kartan.

Reply:
Ah I see that now. Ok. Will update that row